docs/backend-requests/making/jwt-templates.mdx

```diff
@@ -160,7 +160,74 @@ For example, you could use interpolation to build the user's full name:
 }
 ```
 
-Interpolated shortcode substituted values will always be casted to strings. For example, if the user does not have a last name associated, the above full name value would be "null John".
+Interpolated shortcodes will always result to string values. For example, if the user does not have a last name associated, the above full name value would be "null John".
+
+### Coalescing
+
+Shortcode coalescing is a conditional expression of the form:
+
+```json
+{
+  "key": "{{ <operand_1> || <operand_2> || <operand_n> }}"
+}
+```
+
+For example, given a user that:
+
+- has verified their phone number
+- has not verified their email
+- has not provided their first or last name
+- does not have any public or unsafe metadata assigned
+
+and given the following template (comments added for clarity):
+
+```json
+{
+  "has_verified_contact_info": "{{user.has_verified_email || user.has_verified_phone}}",
+
+  // fallback to a string value
+  "full_name": "{{user.full_name || 'Awesome User'}}",
+
+  // fallback to a number value
+  "age": "{{user.public_metadata.age || user.unsafe_metadata.age || 30 }}"
+}
+```
+
+the output after substitution would be similar to the following:
+
+```json
+{
+  "has_verified_contact_info": true,
+  "full_name": "Awesome User",
+  "age": 30
+}
+```
+
+Coalescing is often used to substitute a default fallback value for shortcodes that would otherwise result in `null` or `false` values.
+
+In a coalescing expression, operands are separated by `||`. The result of a coalescing expression is the first operand that is not `null`/`false`. If all operands of the expression are `null`/`false`, the last operand is returned no matter its return value.
+
+Coalescing can be combined with interpolation:
+
+```json
+{
+  "title": "Awesome {{user.first_name || user.last_name || 'user'}}"
+}
+```
+
+Assuming a user that has no first or last name set in their profile, the above would output:
+
+```json
+{
+  "title": "Awesome user"
+}
+```
+
+The rules that sum up how coalescing works are:
+
+- string operands should use single quotes (`'`)
```

Comment:
Team is this about string operants or string literals?

Reply:
String literals, good catch, fixing.